lib/json.h

```diff
@@ -30,7 +30,7 @@ SUPPRESS_WARNING_CLANG_PUSH("-Wformat")
 SUPPRESS_WARNING_CLANG_PUSH("-Wfloat-conversion")
 
 #define PICOJSON_USE_INT64
-#include <picojson.h>
+#include <picojson.h> // IWYU pragma: export
```

Comment:
This is needed for clang-include-cleaner only (still need to file an upstream bug) - see https://github.com/include-what-you-use/include-what-you-use/issues/1696 for some context why it should not be necessary.